File: main/factorial-trailing-zeros/factorial-trailing-zeros.py
def countFactors(n, k):
    count = 0
    while n > 0:
        n //= k
        count += n
    return count

class Solution:
    def trailingZeroes(self, n):
        """
        :type n: int
        :rtype: int
        """
        return countFactors(n, 5)

File: main/factorial-trailing-zeros/factorial-trailing-zeros.cs
public class Solution {
    public int TrailingZeroes(int n)
        => CountFactors(n, 5);

    private static int CountFactors(int n, int k) {
        var count = 0;
        while (n > 0) count += (n /= k);
        return count;
    }
}
